### DISCUSSION TITLE
Where is my money?

### DISCUSSION THREAD

**Discussion by adrien-berchet** [community/community#62990] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,
I received my first sponsor payment recently and it appears properly in the 'Your sponsors' table but I never received the money on my account (and it does not appear on my Stripe account). The payment was made the 11th of July so I think I should already have received it now, right? As far as I can see, my Stripe account is properly configured but maybe I missed something?
Thanks

**Comment by fabricio-fn** [comment]
check the transaction email, see deadlines, transaction data.

veja mais aqui: https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor

**Reply by adrien-berchet** [reply]
Hi :)
Unfortunately I did not received any email about this transaction and I can't find anything helpful for my specific case in the link you gave.

**Reply by Glorycs29** [reply]
Me too, If the problem gets resolved please inform me.

**Reply by adrien-berchet** [reply]
Hi @Glorycs29 
I contacted the support of Github Sponsors and it appeared that, in my case, the payment could just no be processed for an unknown reason... So maybe it's the same in your case and the sponsor should try to make a new sponsorship?

**Reply by Glorycs29** [reply]
Yes, It will take time to process.

**Comment by mehulrajdev** [comment]
Hello! 👋
Thanks for the details. Does the billing apply yearly or monthly?

**Reply by adrien-berchet** [reply]
Hello :)
It's a one time billing with custom amount.
